gnoboard/src/hooks/use-gno.ts

```diff
@@ -1,3 +1,8 @@
+import { SetPasswordRequest } from '@gno/api/gnomobiletypes_pb';
+import { SelectAccountRequest } from '@gno/api/rpc_pb';
+import { CreateAccountRequest } from '@gno/api/rpc_pb';
+import { ListKeyInfoRequest } from '@gno/api/rpc_pb';
+import * as Grpc from '@gno/grpc/client';
 import { GnoAccount } from '@gno/native_modules/types';
 import * as GoBridge from '@gno/utils/bridge';
```

Comment:
Should UseGno use directly `native-modules` instead of `utils/bridge`. That could remove a useless boilerplate interface.